front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -214,6 +219,13 @@ export async function createOrUpgradeAgentConfiguration(
       dataSources: action.dataSources,
     });
   }
+  if (action && action.type === "dust_app_run_configuration") {
```

Comment:
I think we can do if/else if/else never here ?

Reply:
What's the syntax for the else never ?